src/site/_includes/partials/author-stack.njk

```diff
@@ -1,37 +1,11 @@
 <div class="flow">
```

Comment:
You mentioned this partial is used in only 1 place. It will be better to remove it then and replace with the shortcode directly, since it is not reused anywhere.

Reply:
It's only used in 4 different file (for the same purpose)

- /src/site/content/en/learn/css/conclusion/index.md
- /src/site/content/en/learn/design/conclusion/index.md
- /src/site/content/en/learn/forms/conclusion/index.md
- /src/site/content/en/learn/pwa/conclusion/index.md

Also, /src/site/_includes/partials/post-next.njk has a similar use of "authors". I think we should keep the stacks, but maybe make some changes so it can be used in both.

Reply:
Plz see => https://github.com/GoogleChrome/web.dev/pull/7287/files#diff-136f5a836acb3ac96fd5d99e3248e9a083705c1751508eccc4c558ae10411319